src/js/components/charts/pens/xAxisBrush.js

```diff
@@ -3,14 +3,13 @@
 import {isEqual} from "lodash"
 import * as d3 from "d3"
 
-import type {Pen} from "../types"
-import type {Span} from "../../../services/BoomClient/types"
+import type {DateSpan, Pen} from "../types"
```

Comment:
The `Span` type was overloaded. In some places it was `[Date, Date]`, in others it was `[Ts, Ts]`. Now `DateSpan` means the former.